src/front-end/Dockerfile

```diff
@@ -7,4 +7,4 @@ RUN npm run back-end:build
 RUN chmod -R 775 /usr/app
 RUN chown -R node:root /usr/app
 EXPOSE 3000
-ENTRYPOINT ["/usr/app/src/front-end/docker-entrypoint.sh"]
\ No newline at end of file
+ENTRYPOINT ["/usr/app/src/front-end/docker-entrypoint.sh"]
```

Comment:
Recommend updating node version of the image

Reply:
There is another PR that isn't a part of this one yet that addresses this. In this PR the only thing that touched this file was adding the new line to the eof.